### PR TITLE
Adding tags!

### PR DESCRIPTION
**Pull request by grantoesterling**
Adding tags to artbot! Heritage info is in tags so removing that old handling

`#? animated` and `#? granto.eth animated` both work!

Plus adding `yarn dev` to the package.json, bc of context switching 🙃

### CODE REVIEW

File: src/Classes/APIBots/utils.ts [hunk 1]
```diff
@@ -118,25 +117,21 @@ async function getOSName(address: string): Promise<string> {
...
 
-function isWallet(msg: string): boolean {
+export function isWallet(msg: string): boolean {
   return msg.startsWith('0x') || msg.endsWith('eth')
 }
 
-function isVerticalName(msg: string): boolean {
+export function isVerticalName(msg: string): boolean {
   return (
     msg === 'curated' ||
```
Comment (asaltste) [inline src/Classes/APIBots/utils.ts:126]: nit: would be nice for a bit of refactoring work if this was just an array of strings and then used one of the built in array function to see if the message equaled any of the array values

Comment (asaltste) [reply src/Classes/APIBots/utils.ts:126]: could also make it a switch with fall through

File: src/Classes/ArtIndexerBot.ts [hunk 2]
```diff
@@ -72,21 +74,21 @@ export class ArtIndexerBot {
...
         `ArtIndexerBot: Building ${projects.length} ProjectBots using: ${this.projectFetch.name}`
       )
       for (let i = 0; i < projects.length; i++) {
         const project = projects[i]
         if (project.invocations === '0') continue
         let bday = project.start_datetime
+        // Only AB projects use vertical names. Other projects (Engine, Collabs, etc) should use the category name
         const collection = this.toProjectKey(
-          collections[`${project.contract_address}-${project.project_id}`]
+          project.vertical?.category_name?.toLowerCase() !== 'collections'
```
Comment (asaltste) [inline src/Classes/ArtIndexerBot.ts:86]: nit: is there an enum value we can use here for the collections string?

File: src/Data/generated/graphql.ts [hunk 3]
```diff
@@ -10848,23 +10848,23 @@ export type Webflow_Spectrum_Articles_Stream_Cursor_Value_Input = {
   webflow_item_id?: InputMaybe<Scalars['String']>;
```
Comment (asaltste) [inline src/Data/generated/graphql.ts:10848]: I think we want the generated folder included in the .gitignore unless we need it for the discord bot to build.

**Review by asaltste — APPROVED** [review]
Some nits about refactor work but lgtm

**Review by mchrupcala — APPROVED** [review]

File: src/Data/generated/graphql.ts (hunk 3 above)
Comment (asaltste) [reply src/Data/generated/graphql.ts:10848]: ahh cool thanks for the context

**Pull request merged**